packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -1967,7 +1967,10 @@ export const TransactionTable = forwardRef((props, ref) => {
       afterSave(() => {
         const transactions = latestState.current.transactions;
         const idx = transactions.findIndex(t => t.id === id);
-        const parent = transactionMap.get(transactions[idx]?.parent_id);
+        const parentIdx = transactions.findIndex(
+          t => t.id === transactions[idx]?.parent_id,
+        );
+        const parent = transactions[parentIdx];
```

Comment:
nit: you could just use `.find` directly if the index isn't used anywhere else.

Reply:
good call - fixed!